src/test/java/uk/nhs/hee/tis/trainee/reference/TisTraineeReferenceApplicationTest.java

```diff
@@ -27,8 +27,8 @@
 @SpringBootTest
 class TisTraineeReferenceApplicationTest {
 
-  @Test
-  void contextLoads() {
-
-  }
+//  @Test
```

Comment:
Uncomment this, sonarcloud can be configured to ignore coverage of the application class but the test is still needed.